src/components/Common/UI/ProgressBreadcrumbs/ProgressBreadcrumbsTypes.ts

```diff
@@ -0,0 +1,62 @@
+import type { OnEventType } from '@/components/Base/useBase'
+import type { EventType } from '@/types/Helpers'
+
+export interface Breadcrumb {
+  /**
+   * Unique key for the breadcrumb step
+   */
+  key: string
+  /**
+   * Translation key for the breadcrumb label
+   */
+  label: string
+  /**
+   * Optional translation namespace
+   */
+  namespace?: string
+  /**
+   * Optional variables for the breadcrumb label
+   */
+  variables?: Record<string, unknown>
```

Comment:
For adapters, my recommendation here would be to just supply this with a `label: ReactNode` prop and get rid of the `namespace`/`variables` props, and remove translations from the breadcrum altogether. We can supply the translations on our end to the component instances. That way we can manage the translations completely on our end and the partner just knows their adapter needs to render content